packages/grid/_modules_/grid/constants/eventsConstants.ts

```diff
@@ -1,51 +1,51 @@
 // Web standard events
-export const RESIZE = 'resize';
-export const CLICK = 'click';
-export const MOUSE_HOVER = 'mouseover';
-export const FOCUS_OUT = 'focusout';
-export const KEYDOWN = 'keydown';
-export const KEYUP = 'keyup';
-export const SCROLL = 'scroll';
-export const DRAGEND = 'dragend';
+export const GRID_RESIZE = 'resize';
+export const GRID_CLICK = 'click';
+export const GRID_MOUSE_HOVER = 'mouseover';
+export const GRID_FOCUS_OUT = 'focusout';
+export const GRID_KEYDOWN = 'keydown';
+export const GRID_KEYUP = 'keyup';
+export const GRID_SCROLL = 'scroll';
+export const GRID_DRAGEND = 'dragend';
 
 // XGRID events
-export const COMPONENT_ERROR = 'componentError';
-export const UNMOUNT = 'unmount';
-export const GRID_FOCUS_OUT = 'gridFocusOut';
-export const CELL_CLICK = 'cellClick';
-export const CELL_HOVER = 'cellHover';
-export const ROW_CLICK = 'rowClick';
-export const ROW_HOVER = 'rowHover';
-export const ROW_SELECTED = 'rowSelected';
-export const SELECTION_CHANGED = 'selectionChange';
-export const COLUMN_HEADER_CLICK = 'columnClick';
-export const COLUMN_HEADER_HOVER = 'columnHeaderHover';
-export const PAGE_CHANGED = 'pageChange';
-export const PAGESIZE_CHANGED = 'pageSizeChange';
-export const COLUMN_FILTER_BUTTON_CLICK = 'colFilterButtonClick';
-export const COLUMN_MENU_BUTTON_CLICK = 'colMenuButtonClick';
-
-export const SCROLLING_START = 'scrolling:start';
-export const SCROLLING = 'scrolling';
-export const SCROLLING_STOP = 'scrolling:stop';
-
-export const COL_RESIZE_START = 'colResizing:start';
-export const COL_RESIZE_STOP = 'colResizing:stop';
-
-export const COL_REORDER_START = 'colReordering:dragStart';
-export const COL_REORDER_DRAG_OVER_HEADER = 'colReordering:dragOverHeader';
-export const COL_REORDER_DRAG_OVER = 'colReordering:dragOver';
-export const COL_REORDER_DRAG_ENTER = 'colReordering:dragEnter';
-export const COL_REORDER_STOP = 'colReordering:dragStop';
-
-export const ROWS_UPDATED = 'rowsUpdated';
-export const ROWS_SET = 'rowsSet';
-export const ROWS_CLEARED = 'rowsCleared';
-export const COLUMNS_UPDATED = 'columnsUpdated';
-
-export const SORT_MODEL_CHANGE = 'sortModelChange';
-export const FILTER_MODEL_CHANGE = 'filterModelChange';
-
-export const STATE_CHANGE = 'stateChange';
-
-export const MULTIPLE_KEY_PRESS_CHANGED = 'multipleKeyPressChange';
+export const XGRID_COMPONENT_ERROR = 'componentError';
+export const XGRID_UNMOUNT = 'unmount';
+export const XGRID_FOCUS_OUT = 'gridFocusOut';
+export const XGRID_CELL_CLICK = 'cellClick';
```

Comment:
That should also work in data-grid

Reply:
I'm opened to suggestions on how to name these constants - the problem is that some of them are already annotated with `GRID_` and if removed there will be duplicate names. That's why some of them are with `GRID` and some with `XGRID`.

Reply:
so why not adopt `GRID_` everywhere?

Reply:
The same question as Damien, seeing XGrid seems counter-intuitive to me. This variable seems to be a shared one.

I think that it's only once we start better isolating data-grid and x-grid features, that we can have a discussion about Grid vs. XGrid as a prefix.

Reply:
No worries, I'll change it. I would also need to rename the ones that will have clashing names.

Reply:
Renamed